<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_model">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_starter">
       <attribute name="title">
        <string>Starter</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_starter"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_data">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_data"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_control">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_control"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_forecast">
       <attribute name="title">
        <string>Forecast</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_forecast"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_output">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_output"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionDefaultModel"/>
    <addaction name="actionSetCurrentDefault"/>
    <addaction name="actionSetDefaultDir"/>
    <addaction name="separator"/>
    <addaction name="actionOpenModel"/>
    <addaction name="actionSaveModel"/>
    <addaction name="actionSaveModelAs"/>
    <addaction name="separator"/>
    <addaction name="actionViewInfo"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowStartScreen"/>
    <addaction name="actionShowToolbar"/>
    <addaction name="actionShowStarter"/>
    <addaction name="actionShowData"/>
    <addaction name="actionShowControl"/>
    <addaction name="actionShowForecast"/>
    <addaction name="actionShowOutput"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutModel"/>
    <addaction name="separator"/>
    <addaction name="actionAboutInterface"/>
    <addaction name="actionAboutSS"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuModel"/>
   <addaction name="menuView"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDefaultModel">
   <property name="text">
    <string>Load default model</string>
   </property>
  </action>
  <action name="actionSetCurrentDefault">
   <property name="text">
    <string>Set current as default</string>
   </property>
  </action>
  <action name="actionSetDefaultDir">
   <property name="text">
    <string>Set default dir</string>
   </property>
  </action>
  <action name="actionOpenModel">
   <property name="text">
    <string>Open existing model</string>
   </property>
  </action>
  <action name="actionSaveModel">
   <property name="text">
    <string>Save current model</string>
   </property>
  </action>
  <action name="actionSaveModelAs">
   <property name="text">
    <string>Save model as . . .</string>
   </property>
  </action>
  <action name="actionViewInfo">
   <property name="text">
    <string>View model info</string>
   </property>
  </action>
  <action name="actionShowStartScreen">
   <property name="text">
    <string>Start up screen</string>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionShowStarter">
   <property name="text">
    <string>Starter</string>
   </property>
  </action>
  <action name="actionShowData">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionShowControl">
   <property name="text">
    <string>Control</string>
   </property>
  </action>
  <action name="actionShowForecast">
   <property name="text">
    <string>Forecast</string>
   </property>
  </action>
  <action name="actionShowOutput">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run Stock Synthesis</string>
   </property>
  </action>
  <action name="actionAboutModel">
   <property name="text">
    <string>About this model</string>
   </property>
  </action>
  <action name="actionAboutInterface">
   <property name="text">
    <string>About this interface</string>
   </property>
  </action>
  <action name="actionAboutSS">
   <property name="text">
    <string>About Stock Synthesis</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
